src/pages/home/report/ReportActionCompose.js

```diff
@@ -650,14 +749,21 @@ class ReportActionCompose extends React.Component {
             return;
         }
 
-        if ((e.key === CONST.KEYBOARD_SHORTCUTS.ENTER.shortcutKey || e.key === CONST.KEYBOARD_SHORTCUTS.TAB.shortcutKey) && this.state.suggestedEmojis.length) {
+        const suggestionsExist = this.state.suggestedEmojis.length > 0 || this.state.suggestedMentions.length > 0;
+
+        if ((e.key === CONST.KEYBOARD_SHORTCUTS.ENTER.shortcutKey || e.key === CONST.KEYBOARD_SHORTCUTS.TAB.shortcutKey) && suggestionsExist) {
```

Comment:
Coming from https://github.com/Expensify/App/issues/19624 ✋

Not a regression. We want to skip the selection when pressing Shift + Enter as Slack does.